sql/core/src/main/scala/org/apache/spark/sql/execution/joins/CartesianProductExec.scala

```diff
@@ -60,23 +61,13 @@ class UnsafeCartesianRDD(
 case class CartesianProductExec(
     left: SparkPlan,
     right: SparkPlan,
-    condition: Option[Expression]) extends BinaryExecNode {
+    joinType: JoinType,
```

Comment:
CartesianProduct is always an inner join.

Reply:
Thanks for review!  Addressed in 4397ce88a99b4777067e7a636f12e22612b04be5.